cypress/support/fragments/inventory/search/browseCallNumber.js

```diff
@@ -90,4 +93,13 @@ export default {
       }),
     );
   },
+
+  checkValueInRowAndColumnName(indexRow, columnName, value) {
+    cy.expect(
+      resultList
+        .find(MultiColumnListRow({ index: indexRow }))
+        .find(MultiColumnListCell({ column: columnName }))
```

Comment:
Can we modify this part like so? to not use `MultiColumnListRow`
`MultiColumnListCell({
        row: indexRow,
        column: columnName,
      })`